apps/website/src/pages/about/tech/commands.tsx

```diff
@@ -70,6 +71,8 @@ const signature = (command: NamedCommand) => {
 };
 
 const AboutTechPage: NextPage = () => {
+  const { copyToClipboard } = useClipboard();
+  const [index, setIndex] = useState<string>("");
```

Comment:
Can we give this a name that more clearly represents what this state is? Like `selectedCommand / setSelectedCommand`?